### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by cayter** [chakra-ui/ark#717] [Q&A]
Will there be more components like DatePicker, Calendar and DatePicker?

**Comment by anubra266** [comment]
Yes there will be

**Reply by cayter** [reply]
Thanks for the quick reply, wanted to also check: is this library gonna be like chakra-ui selling a pro version?

**Reply by anubra266** [reply]
@cayter can't say for sure yet

**Reply by vitalijalbu** [reply]
Will be also autocomplete component??

**Reply by anubra266** [reply]
@vitalijalbu I suppose you mean a combobox. Yes that's available already

**Comment by vitalijalbu** [comment]
Will Ark replace chakra ui in the future?? Just read this: [https://twitter.com/thesegunadebayo/status/1643706973341335553](https://twitter.com/thesegunadebayo/status/1643706973341335553)

**Comment by vitalijalbu** [comment]
Can we suggest all components we want on the roadmap page? 👀
